boards/st/nucleo_h563zi/doc/index.rst

```diff
@@ -269,6 +288,10 @@ This can be done by executing the following commands.
    $ pyocd pack --install stm32h5
 
 
+Alternatively, this interface will be supported by a next openocd version.
+When available, OpenOCD could be used. Same process applies with other tools.
+
```

Comment:
Unclear

Reply:
replaced by "_Alternatively, the openocd interface will be supported by a next openocd version.
When available, OpenOCD could be used in the same way as other tools._"